src/main/data/facilities.json

```diff
@@ -625,6 +625,16 @@
     "code": "BF8",
     "description": "Buffalo 8 Post Production"
   },
+  {
+    "code": "BFP",
+    "contact": {
+      "address": "Caracas, Miranda, Venezuela",
```

Comment:
I would remove this incomplete address

Reply:
The address they provided was: Sta Eduvigis Av Luis Guillermo Villegas Caracas (Venezuela). When that address is placed in Google Maps it returns: Caracas, Miranda, Venezuela as the address. That’s the address I entered into the registry.


Jerry

Reply:
The address from Google came out: C. Luis Guillermo Villegas Blanco, Caracas, Miranda, Venezuela 

Pull request updated